pkg/frontend/openshiftcluster_putorpatch.go

```diff
@@ -149,9 +149,10 @@ func (f *frontend) _putOrPatchOpenShiftCluster(ctx context.Context, log *logrus.
 	// Patch should be used for updating individual fields of the document.
 	case http.MethodPatch:
 		ext = converter.ToExternal(doc.OpenShiftCluster)
-		converter.ExternalNoReadOnly(ext)
 	}
 
+	converter.ExternalNoReadOnly(ext)
```

Comment:
This change seems odd at a glance, but maybe it is correct - is the intended behavior that readonly fields shouldn't be returned on GET either, or am I reading this incorrectly? I could see this affecting things that need to read SystemData/etc